README.md

```diff
@@ -2,32 +2,36 @@
+Welcome to a custom **Copilot** workshop prepared by the **GitHub’s Expert Service** team!
 
-This workshop involves several technology stacks, therefore it is essential that you need to meet prerequisites as outlined in the prerequisites getting started if you want to follow along. Otherwise, please follow along and listen to what the GitHub Expert Service team's instructor demo to see how it works in practice.
+This workshop involves several technology stacks, therefore it is essential that you need to meet prerequisites as outlined in the prerequisites getting started if you want to follow along. Otherwise, please follow along and listen to the GitHub Expert Service team's instructor demo to see how it works in practice.
+
+> :blue_book: **NOTE**: This workshop will be conducted through **Codespace**, but you can find a full step-by-step self-paced guide that you can try later without Codespace:
+[**gh.io/octofit-tutorial**](https://gh.io/octofit-tutorial)
 
 ## Objectives and key takeaways
 
 ### Objectives
 
-- Walk through a real-world use case:
-  - Building a fitness tracker app from end to end.
-  - Create a full stack of technologies for the infrastructure, frontend, and backend.
-  - Using the latest GitHub Copilot features.
+**Walk through a real-world use case**:
+  - Building a fitness tracker app from end to end
+  - Create a full stack of technologies for the infrastructure, frontend, and backend
+  - Using the latest **GitHub Copilot** features
 
 ### Key takeaways
 
-- Build a full stack application with GitHub Copilot using exercises and instructions that you can take with you
-- Experience the coolest, latest GitHub Copilot features, including Copilot and Copilot Chat
-- Learn different concepts in GitHub Copilot and best practices for effectively leveraging Copilot in a variety of scenarios
+- Build a full stack application with **GitHub Copilot** using exercises and instructions that you can take with you
+- Experience the coolest, latest GitHub Copilot features, including **Copilot** and **Copilot Chat**
+- Learn different concepts in **GitHub Copilot** and best practices for effectively leveraging **Copilot** in a variety of scenarios
 
 Without further ado, let's get started.
 
-## Table of Contents
+## Table of Contents (Total Estimated Time: About 30 minutes)
 
-1. [The OctoFit Tracker app story](docs/1_Story/README.md)
-2. [Prerequisites and asetup of the development environment](docs/2_Prerequisites/README.md)
-3. [Getting started - app frontend and backend creation](docs/3_GettingStarted/README.md)
-4. [The OctoFit Tracker database and app backend creation](docs/4_BackendSettings/README.md)
-5. [Populate the database with sample data](docs/5_PopulateDBwData/README.md)
-6. [Using the Codespace endpoint to access the Django REST Framework](docs/6_CodespaceDjangoRESTFramework/README.md)
+1. [The OctoFit Tracker app story](docs/1_Story/README.md) - :alarm_clock: Estimated Time: 2 minutes
```

Comment:
I rather not put times on these. I don't think that is necessary

Reply:
Estimated time?

Reply:
Removed